packages/core/src/components/popover/popoverSharedProps.ts

```diff
@@ -142,6 +142,13 @@ export interface IPopoverSharedProps extends IOverlayableProps, IProps {
      */
     usePortal?: boolean;
 
+    /**
+     * The container that the popover renders its contents to.
+     * This prop only available when `usePortal` is set to `true`
+     * @default document.body
+     */
+    container?: HTMLElement;
```

Comment:
this shouldn't be necessary since you added it to `IOverlayableProps` which is extended here.